doc/base/classes.xml

```diff
@@ -2556,7 +2558,7 @@
 			<argument index="2" name="weight_scale" type="float" default="1.0">
 			</argument>
 			<description>
-				Add a new point at the given position [code]pos[/code] with the given identifier [code]id[/code]. The [code]weight_scale[/code] has to be 1 or larger.
+				Adds a new point at the given position with the given identifier. A point with a lower [code]weight_scale[/code] will be prefered to form a path. The [code]id[/code] must be positive and the [code]weight_scale[/code] must be 1 or larger.
```

Comment:
Avoid the passive voice: `will be prefered` -> `Prefers points with a lower weight_scale` or `Prioritizes points with...`.
The function is abstract. Could you add a code example maybe?